### DISCUSSION TITLE
Welcome to TopoPyScale Discussions!

### DISCUSSION THREAD

**Discussion by ArcticSnow** [ArcticSnow/TopoPyScale#61] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joelfiddes** [comment]
This is cool and great replacement for slack with the 3mnth wipeout thing! Can organize discussion topics by category it seems

**Reply by ArcticSnow** [reply]
Totally. I was struggling with all the emails lately. ;) 

Very fortunate I stumbled across this in the settings. @ealonsogzl @yelizy @paswyss @krisaalstad and others, please come here to post anything related to TopoPyScale. No more emails 🎉. This way, all is tied together with the code. 
Looking forward to push development together on this toolbox and hear about your applications!
Cheers
